### ISSUE TITLE
Add examples to the documentation

### ISSUE THREAD

**Issue by lastmjs** [scramjs/express-web-components#14]
Think of the best way to add examples.

**Comment by lastmjs** [comment]
We need to add examples of most of the common Express use-cases:

- [ ] Creating an app and listening on a port
- [ ] Adding a generic middleware (app.use)
- [ ] Adding a middleware with a method (app.all, app.get, app.post, etc.)
- [ ] Doing configurations (all other app or express method calls that require a config element)
- [ ] Adding a middleware with a method to router: router.method (router.all, router.get, router.post, etc.)
- [ ] Adding generic middleware to router (router.use)
- [ ] router.route
- [ ] Configuring router (router.param)
- [ ] app.route and router.route

To summarize, show everything that you can do. app middleware, router middleware, and app and router configuration, properties, and methods
